app/(main)/page.tsx

```diff
@@ -417,24 +445,28 @@ export default function Home() {
     localStorage.setItem("publishedUrl", url);
   };
 
-  if (loading) {
+  if (loading || isLoading) {
```

Comment:
why are we maintaining loading as well as isLoading variable?

Reply:
loading was there previously to load state from local storage, set state, save state to local storage.
isLoading is coming from useUser() hook and is an intermediate state to know whether the user is authenticated or not.